### DISCUSSION TITLE
关于解盘申请

### DISCUSSION THREAD

**Discussion by SylarLong** [SylarLong/iztro#124] [Announcements]
之前让大家把解盘申请提到了issue里，现在想想不是特别合适，所以决定把解盘申请移动到discusstions里面来。有解盘需求的朋友可以 [点击此处](https://github.com/SylarLong/iztro/discussions/new?category=%E8%A7%A3%E7%9B%98%E7%94%B3%E8%AF%B7) 创建一个解盘申请帖子。

帖子内需包含下列信息：

- `星盘信息` 或 `出生日期/时间/性别`

  星盘信息可以是一张图片，可以到 http://ziwei.pub/astrolabe 上面生成以后把出生时间等敏感信息码住以后贴出来，如果不介意的朋友也可以直接贴链接或者 `出生日期/时间/性别`，需要注明是 `农历` 还是 `阳历`。

- 时间是否准确

  时间的准确性直接影响了解盘的准确性。所以请一定确认自己的出生时辰。Ref. [为什么星盘显示的个性和我实际个性差那么多？](https://github.com/SylarLong/iztro/discussions/122)

- 咨询事项

  咨询事项一般以个性推演，与人相处模式，工作观，感情观，消费观等为主。

  - ❌以下是不推荐的提问方式

    - 我将来会有多少钱？
    - 我会有几个孩子？
    - 我将来会结几次婚？
    - ...

  - ✔以下是推荐的提问方式

    - 我将来和伴侣相处得怎样？
    - 我的小孩是个什么个性？
    - 我适合什么样的工作？
    - 我适合做生意吗？
    - 我将来的另一半是什么样的个性？
    - ...
---

通常我都会先进行个性推演，如果个性推演匹配率高，再进行具体事项的追问，如果个性推演和你本身个性差异较大，请选择终止解盘或者确定出生时。

**Comment by marcushsu** [comment]
敝人也从事紫微研究及IT相关工作均有二、三十年之久，误打误撞来到这里，对于提供解说服务这件事有个建议。
在我们进行八字、紫微等解盘之前，有个很重要的工作叫定盘。如果盘不对，接下来的解说都是白说，而且弄不好还会造成问事者的心理负担。先不论时辰交替附近出生的人一定需要先定盘，在我数十年算命的经验里，记错时辰（包括当事人父母）的占比没有三四成也有两三成，尤其是出生年份越早、出生地点越乡下的当事人，其出生日期错误的可能性越大。还有阳历、阴历不分的。而我推命已经可以从对方过去发生的事情，几乎百分百认定他是否记错生日，或者从前后时辰来推断并告诉对方记错时辰，大概有九成的结果是对方在更积极地求证后，惊讶地反馈他们的确记错了。将确认时辰的责任交给问事者其实是一个谬误，因为他们常会固执己见、认为自己从小所记的时辰没有错，甚至连他们的父母都会坚持自己没记错。因此，在未定盘的情况下即为问事者解说，其实非常不妥，极有可能给予对方错误的指引，甚至造成无谓的心理负担。这个与是否收费无关，而是我们一个有追求的论命者应有的基本伦理原则。单就个性来推演并不够，因为个性的描述常常是模拟两可，尤其遇到双星的人，其个性常有里外、明暗之分，就算当事人自己也不见得弄得很清楚。我们除了个性之外还会直接断事，譬如某年是否曾经发生车祸或动手术、父母是否已离异、某年是否有官司甚至牢狱、某年是否工作有重大变动或者被裁员等，起码要论中三个以上才能定盘，进而解盘。

**Reply by SylarLong** [reply]
非常赞同，感谢补充

**Reply by SylarLong** [reply]
@marcushsu 非常感谢老师的提醒与指正，我已停止了解盘活动

**Reply by wangzelin16** [reply]
请问定盘，解盘么？

**Comment by SylarLong** [comment]
此活动已关闭 Ref. https://github.com/SylarLong/iztro/discussions/151
